stubs/redis/redis/cluster.pyi

```diff
@@ -149,6 +150,7 @@ class NodesManager:
         lock: Lock | None = None,
         dynamic_startup_nodes: bool = True,
         connection_pool_class: type[ConnectionPool] = ...,
+        address_remap: Callable[[str, int], tuple[str, int]] | None = None,
```

Comment:
You also need to add the attribute annotation to the class: https://github.com/redis/redis-py/compare/v4.5.4...v4.5.5#diff-a324b052f8b1189052b9606664591ba6bfe7669e971ddd027093413bc24bd372R1300